dataminer-overview/Pricing/Professional_service_credits.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: Professional_service_credits
+---
+
+# Professional Service credits
+
+Agility and adaptability are at the core of DataMiner. Skyline applies the same principles to its professional services: **maximum agility** with value and budget control in mind. Instead of buying a certain number of certificates or engineering hours, DataMiner users can order a single product – Skyline professional services credits – which can be used freely towards any of Skyline's professional services, when you need them and match your priorities.
+
+Below, you can find an overview of how the Service Credits can be used.
+
+> [!TIP]
+> If you have any questions, take a look at the [FAQ related to Professional Service credits](xref:FAQs_Pricing#professional-service-credits).
+
+## Engineering
+
+| Item | Details |
+|------|---------|
+| Integration | 4 credits<br>Per hour, per person |
+| Consultancy | 5 credits<br>Per hour, per person |
+
+## Certifications
+
+| Item | Details |
+|------|---------|
+| Fundamentals Certificate | 16 credits<br>Per certificate |
+| Configurator Certificate | 36 credits<br>Per certificate |
+| Visual Overview | 6 credits<br>Per certificate |
+| Automation Certificate | 24 credits<br>Per certificate |
+| Connector Integration: SNMP Basics | 40 credits<br>Per certificate |
+| Connector Integration: HTTP Basics | 40 credits<br>Per certificate |
+
+## Dedicated training
+
+| Item | Details |
+|------|---------|
+| DevOps Enablement Service | 5 credits<br>Per hour, per person |
+| [Open Day Training](https://community.dataminer.services/open-training/) | 14 credits<br>Per day, per person |
+| Fundamentals Training (2 days) | 160 credits<br>Per session |
+| Configurator Training (2 days) | 160 credits<br>Per session |
+| Sys Admin Training (1 day) | 90 credits<br>Per session |
+| Agile Fundamentals Training | 40 credits<br>Per session |
+| Kanban Workshop | 40 credits<br>Per session |
+
+> [!NOTE]
+>
+> - Except for Agile Awareness and DevOps Enablement, all training is available in both in-person and online format. For in-person training, additional credits are required to cover the transportation and on-site expenses of having the instructor present. These expenses are not included in the course fees.
```

Comment:
Agile Awareness is not mentioned in the table above. Should I change this to something else, or is something missing?